EffortlessUI-new/public/base.css

```diff
@@ -0,0 +1,384 @@
+}
+.primary-button.lightOrange {
+  background-color: var(--lightOrange);
+}
+.primary-button.yellow {
+  background-color: var(--yellow);
+}
+.primary-button.lightgreen {
+  background-color: var(--lightgreen);
+}
+.primary-button.orange {
+  background-color: var(--orange);
+}
+.primary-button.grey {
+  background-color: var(--grey);
+}
+.primary-button.maroon {
+  background-color: var(--maroon);
+}
+
+/* icon button */
+.icon-button {
+  cursor: pointer;
+  background-color: var(--grey);
+  border: none;
+  padding: 0.5rem;
+}
+
+.icon-button > img {
+  height: 30px;
+  width: 30px;
+}
+
+.transparentBtn {
+  cursor: pointer;
+  padding: 0.2rem 1rem;
+  background-color: white;
+  color: black;
+  border: 2px solid var(--grey);
+  outline-style: none;
+}
+
+/* social media icon */
+.socialIcon > img {
+  height: 3rem;
+  width: 3rem;
+}
+
+/* Avatar */
+.faceavatar {
+  height: 50px;
+  width: 50px;
+  border-radius: 50%;
+}
+
+.avatar-alphabet {
+  width: 3rem;
+  height: 3rem;
+  color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  font-size: 2rem;
+  align-content: center;
+  font-weight: 900;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+/* alert */
+.alert {
+  display: flex;
+  align-items: center;
+}
+
+.alertText {
+  background-color: white;
+  border: 1px solid black;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  width: 20rem;
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem 2rem 1rem 0.5rem;
+  align-items: center;
+}
+
+.alertText.success {
+  color: green;
+}
+.alertText.failure {
+  color: red;
+}
+.alertText.warning {
+  color: #d97706;
+}
+.alertText.info {
+  color: var(--blue);
+}
+
+.alertIcon {
+  height: 42px;
+  padding: 0.3rem 0.5rem;
+}
+
+.alertIcon.success {
+  background-color: #42b72a;
+}
+.alertIcon.failure {
+  background-color: rgb(179, 11, 11);
+}
+.alertIcon.warning {
+  background-color: #d97706;
+}
+.alertIcon.info {
+  background-color: var(--blue);
+}
+
+.alert > div > img {
+  width: 40px;
+  height: 40px;
+}
+
+.cross {
+  cursor: pointer;
+}
+
+@media screen and (max-width: 520px) {
+  .alertText {
+    width: 14rem;
+  }
+}
+
+/* Navbar */
+.nav1__navList {
+  max-width: 100%;
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #111827;
+  color: white;
+  padding: 1rem;
+}
+
+.nav1__navEnd {
+  display: flex;
+}
+
+.nav1__navStart > a {
+  margin-right: 1rem;
+  text-decoration: none;
+  color: inherit;
+}
+
+.nav1__navEnd > span {
+  margin-right: 1rem;
+  display: flex;
+  align-items: center;
+}
+
+.nav1__navEnd > span > button {
+  padding: 0.5rem 1.5rem;
+  border-radius: 0.5rem;
+  border: 1px solid white;
+  color: white;
+  background-color: #111827;
+}
+
+.nav1__navEnd > span > button:hover {
+  background-color: white;
+  color: black;
+  cursor: pointer;
+}
+
+@media screen and (max-width: 564px) {
```

Comment:
You can have all your media queries grouped together as per sections or category, Later it will help you in refactoring.